sources/ui/src/main/kotlin/com/sedsoftware/blinktracker/ui/camera/core/VisionImageProcessor.kt

```diff
@@ -38,7 +53,27 @@ class FaceDetectorProcessor(detectorOptions: FaceDetectorOptions) : VisionImageP
             return
         }
 
-        detector.process(InputImage.fromMediaImage(image.image!!, image.imageInfo.rotationDegrees))
+        // Get average luminosity and if light is too dim, brighten image
+        var inputImage = InputImage.fromMediaImage(image.image!!, image.imageInfo.rotationDegrees)
+        val currentTimestamp = System.currentTimeMillis()
+        if (currentTimestamp - lastAnalyzedTimestamp >=
+            TimeUnit.SECONDS.toMillis(1)) {
+            val buffer = image.planes[0].buffer
+            val data = buffer.toByteArray()
+            val pixels = data.map { it.toInt() and 0xFF }
```

Comment:
0xFF needs its own constant to pass detekt check I think
`This expression contains a magic number. Consider defining it to a well named constant. [MagicNumber]`